apps/frontend/core-dapp/src/features/deposit/DepositStore.ts

```diff
@@ -119,4 +130,17 @@ export class DepositStore {
 
     return this.root.baseTokenStore.formatUnits(depositAmount)
   }
+
+  get ppoReward(): string | undefined {
+    const { depositFeesBN } = this
+
+    // If there's no fee, there's no PPO reimbursement
+    if (depositFeesBN === undefined || depositFeesBN.eq(0)) return undefined
```

Comment:
And the return 0 here?

```suggestion
    if (depositFeesBN === undefined || depositFeesBN.eq(0)) return '0'
```